spec/system/nsm/view_claim_spec.rb

```diff
@@ -97,11 +97,19 @@
   end
 
   it 'show the work items and summary' do
-    visit work_items_nsm_steps_view_claim_path(claim.id)
+    visit claimed_costs_work_items_nsm_steps_view_claim_path(claim.id)
 
     # items
     expect(all('table caption, table td, table th').map(&:text)).to eq(
       [
+        'Cost summary', 'Item', 'Net cost', 'VAT', 'Total',
```

Comment:
I'm not sure that this is adding anything to the test, which is about claimed work items. Rather than include all of this new extra stuff in every expectation, given it seems kinda irrelevant to the test, perhaps we could change how we do the test from an `eq` to an `include`? Or change the selector string from `'table caption, table td, table th'` to something that just captures the bits we're interested in?

Reply:
`eq` to `include` makes sense, will do